app/controllers/security_group_controller.rb

```diff
@@ -211,7 +211,7 @@ def update_finished
     td = session[:security_group][:task]
     task = MiqTask.find(td[:id])
     if MiqTask.status_ok?(task.status)
-      add_flash(_("#{td[:resource]} #{td[:action]}d"))
+      add_flash("#{td[:resource]} #{td[:action]}d")
```

Comment:
ouch :) Should we add a TODO comment/issue or something like that to list the relevant actions and do a hash lookup?

Reply:
I created https://github.com/ManageIQ/manageiq-ui-classic/issues/5812

For now, I need to remove `_()` from there, because specs won't pass as long as there is `_("... #{} ...")` in catalog.